scp/scp.go

```diff
@@ -41,6 +41,11 @@ func Launch(cfg *config.Config, env config.Environment, instance aws.EC2Result,
 		return errors.New("missing `ssh-user` in config file")
 	}
 
+	// Set AWS_PROFILE when using new AWS environments
+	if contains(config.AwsbEnvs, env.Profile) {
```

Comment:
Consider assigning this to a var as you're using it multiple times and its unnecessary work to evaluate this expression again.

You could also add this as a method on the `Config` struct  as its used in multiple packages. I think this makes more sense as you're asking the config to describe its self so it makes sense for it to encapsulate that logic and let the user just call the method.